homework/index.js

```diff
@@ -28,26 +22,139 @@
     });
     return elem;
   }
+  function formatDate(dateString) {
```

Comment:
Place a blank line between function definitions for improved readability. (Just like you use blank lines to separate paragraphs in a piece of text).